sphinx/themes/basic/static/doctools.js

```diff
@@ -176,10 +176,7 @@ const Documentation = {
    * helper function to focus on search bar
    */
   focusSearchBar : () => {
-    document
-      .querySelectorAll("input[name=q]")
-      .first()
-      .focus()
+    document.querySelectorAll("input[name=q]")[0]?.focus();
```

Comment:
I guess we could also just always assume there is an `input[name=q]` element

Reply:
I wouldn't assume that.
Site authors might have disabled the search box in their setup.